source/agora/consensus/protocol/Nominator.d

```diff
@@ -170,8 +170,8 @@ extern(D):
 
     protected bool prepareNominatingSet (out ConsensusData data) @safe
     {
-        this.ledger.prepareNominatingSet(data, Block.TxsInBlock);
-        if (data.tx_set.length != Block.TxsInBlock)
+        this.ledger.prepareNominatingSet(data, 8);
```

Comment:
There are two `8`s in the `Nominator` class, which feels like the `8` is something special. Should it be constant like `defaultTxsInBlock`? If your big picture is to remove the two 8s in the end, it's OK.

Reply:
This will be replaced with time-based transaction selection. And the maximum limit of the block size is something we need to define later. For example https://github.com/bpfkorea/agora/issues/407